codalab/rest/interpret.py

```diff
@@ -183,6 +186,12 @@ def fetch_interpreted_worksheet(uuid):
     # Fetch items.
     worksheet_info['raw'] = get_worksheet_lines(worksheet_info)
 
+    # Replace searches with raw items.
```

Comment:
Do you think you'll be able to unify this with `fetch_worksheet` from `worksheets.py`?

Reply:
I'm not sure I understand this question, fetch_worksheet only cares about the raw items in the worksheet, whereas this is replacing any raw items that contain search directives with the result of the search? For example, if my worksheet just contained 
"% search .limit=2"
I would not want the worksheet to seem like it actually contained
[bundle1]{0xw50u5oijwed} 
[bundle2]{0xoih0w3523n5}

Reply:
I just meant that there would be one REST endpoint that would return both the (raw) items as well as the blocks.  Otherwise there's some code duplication in two places, and there's inconsistent handling of owner, for example.

Reply:
Ah okay I see what you mean, I'll take a look.

Reply:
There's not a very clean way to unify this, I think this should be separated into its own diff after this is merged.